purchases/api-defauts.txt

```diff
@@ -994,11 +996,15 @@ package com.revenuecat.purchases.models {
   }
 
   @dev.drewhamilton.poko.Poko public static final class GooglePurchasingData.Subscription extends com.revenuecat.purchases.models.GooglePurchasingData {
-    ctor public GooglePurchasingData.Subscription(String productId, String optionId, com.android.billingclient.api.ProductDetails productDetails, String token);
+    ctor public GooglePurchasingData.Subscription(String productId, String optionId, com.android.billingclient.api.ProductDetails productDetails, String token, optional @com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI com.revenuecat.purchases.models.Period? billingPeriod, optional @com.revenuecat.purchases.ExperimentalPreviewRevenueCatPurchasesAPI java.util.List<? extends com.revenuecat.purchases.models.GooglePurchasingData>? addOnProducts);
+    method public java.util.List<com.revenuecat.purchases.models.GooglePurchasingData>? getAddOnProducts();
```

Comment:
Seems we haven't added an experimental annotation for these?

Reply:
Yeah, looks like Poko didn't transitively apply the experimental annotation here. I'll see if I can add it manually by explicitly writing the getter functions.